### ISSUE TITLE
Problems downloading latest Jaybird (Firebird JDBC) driver due to dependency relocation

### ISSUE THREAD

**Issue by mrotteveel** [dbeaver/dbeaver#8294]
The latest version of Jaybird (Firebird JDBC driver) was released with new artifact coordinates on Maven (`org.firebirdsql.jdbc:jaybird` with a version per Java version). The old coordinates - for Java 8 `org.firebirdsql.jdbc:jaybird-jdk18` - now have a POM with a relocation, see the [pom.xml for `org.firebirdsql.jdbc:jaybird-jdk18:4.0.0.java8`](https://repo1.maven.org/maven2/org/firebirdsql/jdbc/jaybird-jdk18/4.0.0.java8/jaybird-jdk18-4.0.0.java8.pom):

```xml
<distributionManagement>
  <relocation>
    <artifactId>jaybird</artifactId>
  </relocation>
</distributionManagement>
```

When updating the driver to version `4.0.0.java8` in DBeaver, DBeaver doesn't follow the relocation specified in the POM, but tries to directly download the jar assuming the standard Maven conventions. This results in the error:

```
Driver file download failed.
Do you want to retry?

Reason:
Can't open 'https://repo1.maven.org/maven2/org/firebirdsql/jdbc/jaybird-jdk18/4.0.0.java8/jaybird-jdk18-4.0.0.java8.jar': Not Found
```

Ideally, DBeaver should follow the relocation specified in the pom.xml to `org.firebirdsql.jdbc:jaybird:4.0.0.java8`. However, if that is not possible, I would recommend that DBeaver switch to the new coordinates. As a complication, the new coordinates of Jaybird has older versions (3.0.x and older) with relocation to the 'old' coordinates.

**Comment by kseniiaguzeeva** [comment]
I cannot reproduce the issue. The path is shown maven:/org.firebirdsql.jdbc:jaybird-jdk18:RELEASE for driver 4.0.0.java8 in DBeaver.
Have you tried:

1. Open Connection settings> Main> Edit Driver Settings
2. Press Download/Update btn
3. Select 4.0.0.java8 in Version dropdown menu.
4. Press Download?

If it doesn't help please tell your OS an DBeaver version.

**Comment by mrotteveel** [comment]
When I reported the issues, I was using DBeaver 7.0.0. I have updated from DBeaver 7.0.0 to 7.0.1, and I also don't get an error anymore, but it also doesn't download the driver.

I'm using the 64 bit version on Windows 10

1. My starting point is (`org.firebirdsql.jdbc:jaybird-jdk18-3.0.8):
![image](https://user-images.githubusercontent.com/5352689/78375821-a8dca780-75cd-11ea-9b95-548d4ed280d0.png)

2. Then I click on Download/Update, and select version 4.0.0.java8
3. Click Download
4. The dialog closes, and no error appears anymore at this time, but attempts to test the connection result in error 
```
Can't create driver instance
  Error creating driver 'Firebird' instance.
Most likely required jar files are missing.
You should configure jars in driver settings.

Reason: can't load driver class 'org.firebirdsql.jdbc.FBDriver'
  Error creating driver 'Firebird' instance.
Most likely required jar files are missing.
You should configure jars in driver settings.

Reason: can't load driver class 'org.firebirdsql.jdbc.FBDriver'
    org.firebirdsql.jdbc.FBDriver
    org.firebirdsql.jdbc.FBDriver
```

If I downgrade to DBeaver 7.0.0, then after step 3 I get error
```
Driver file download failed.
Do you want to retry?

Reason:
Can't open 'https://repo1.maven.org/maven2/org/firebirdsql/jdbc/jaybird-jdk18/4.0.0.java8/jaybird-jdk18-4.0.0.java8.jar': Not Found
```

**Comment by serge-rider** [comment]
I will update maven artifact configuration in the next version. Thanks for th report.

Also there was a bug in jar files resolution (fixed in #8326)

**Comment by kseniiaguzeeva** [comment]
Verified.
